emoji/core.py

```diff
@@ -45,3 +45,56 @@ def replace(match):
             return unicode_codes.EMOJI_UNICODE.get(match.group(1), match.group(1))
 
     return pattern.sub(replace, string)
+
+def demojize(string):
+
+    """Replace unicode emoji in a string with emoji shortcodes. Useful for storage.
+
+    :param string: String contains unicode characters. MUST BE UNICODE.
+
+        >>> import emoji
+        >>> print(emoji.emojize("Python is fun :thumbs_up_sign:"))
+        Python is fun 👍
+        >>> print(emoji.demojize(u"Python is fun 👍"))
+        Python is fun :thumbs_up_sign:
+        >>> print(emoji.demojize("Unicode is tricky 😯".decode('utf-8')))
+        Unicode is tricky :hushed_face:
+    """
+
+    # via Martijn Pieters, http://stackoverflow.com/questions/26568722/remove-unicode-emoji-using-re-in-python
+    if PY2:
```

Comment:
@BrendanAdkins The `u"string"` syntax is supported by both Python 2 and 3 so you can drop the check here and the `else` below.